resources/opentelemetry-kw-telemetry-values.yaml

```diff
@@ -1,9 +1,10 @@
 telemetry:
-  enabled: True
   metrics:
+    enabled: True
     port: 8080
   tracing:
-    otlp/jaeger:
-      endpoint: "my-open-telemetry-collector.jaeger.svc.cluster.local:14250"
+    enabled: True
+    jaeger:
```

Comment:
shouldn't this be otlp/jaeger>

Reply:
The value is `.Values.tracing.jaeger.endpoint`, which translates into the OpenTelemetryCollector CR `exporter.otlp/jaeger`.